Refine known-issues.md (#746)
argonne-lcf/user-guides: docs/aurora/known-issues.md

#### Ping Failures

> Network and compute node instabilities may lead to unaccessible compute nodes. If your job output shows an error message like these:
> 
> ```

Reviewer: ```suggestion
Network and compute node instabilities may lead to inaccessible compute nodes, which will cause MPI ranks on those nodes to become unreachable. If your job output shows an error message like these:

```output
```

ping RPC timeout from x4212c7s0b0n0.hostmgmt2212.cm.aurora.alcf.anl.gov after 120s
```

> ```

Reviewer: ```suggestion
```output
```

ping failed on x4304c1s6b0n0: No reply from x4307c2s6b0n0.hostmgmt2307.cm.aurora.alcf.anl.gov after 87s
```

then most likely your application will crash or hang. If you see these, the best action is to kill the job and re-run it (from the last checkpoint, if there has been one). Potential issues can be discovered when the non-responsive node gets brought back online. Users may query PBS for more info.

> ```
> pbsnodes x4307c2s6b0n0 |grep comment
>      comment = StabilityDB 2025-02-24T04:38:14: hbm_controller_errors
> ```

Reviewer: ```suggestion
```console
$ pbsnodes x4307c2s6b0n0 |grep comment
     comment = StabilityDB 2025-02-24T04:38:14: hbm_controller_errors
```
```

* A large number of Machine Check Events from the PVC, which causes nodes to panic and reboot.
* HBM mode is not automatically validated. Jobs requiring flat memory mode should test by looking at `numactl -H` for 4 NUMA memory nodes instead of 16 on the nodes.

> * Application failures at single-node are tracked in the JLSE wiki/confluence [page](https://apps.cels.anl.gov/confluence/pages/viewpage.action?pageId=4784336)

Reviewer: ```suggestion
* Application failures at the single-node level are tracked in the [JLSE Wiki/Confluence page](https://apps.cels.anl.gov/confluence/pages/viewpage.action?pageId=4784336)
```
